src/App.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable import/extensions */
+import './App.css';
+import { Routes, Route } from 'react-router-dom';
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { getRockets } from './redux/Rockets/rocketReducer';
+import { getMissions } from './redux/Missions/missionsReducer';
+import Navbar from './Components/navbar/Navbar';
+import Missions from './Components/missions/Missions';
+import Profile from './Components/profile/Profile';
+import Rockets from './Components/Rockets/Rockets';
+
+function App() {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getRockets());
+  });
+  useEffect(() => {
+    dispatch(getMissions());
```

Comment:
- [ ] Please fix the moment when the mission data is fetched. The project request to fetch the data when the user navigates to the mission session.

<img width="869" alt="Screen Shot 2022-11-18 at 10 04 34 PM" src="https://user-images.githubusercontent.com/47673311/202826565-65eff328-073b-4ce3-8b24-ce5b030bdaf5.png">